src/renderer/analytics/highlight-search-results.tsx

```diff
@@ -48,20 +39,21 @@ export function highlight(fuseSearchResult: any) {
   };
 
   return fuseSearchResult
-    .filter(({ matches }: any) => matches && matches.length)
-    .map(({ item, matches }: any) => {
+    .filter(
+      ({ matches }: Fuse.FuseResult<LogEntry>) => matches && matches.length,
```

Comment:
```suggestion
      ({ matches }: Fuse.FuseResult<LogEntry>) => Array.isArray(matches),
```

nit: if we want to check if something is a JS Array, use this method directly.